content/articles/secondary-dns-provider-dns-made-easy.markdown

```diff
@@ -0,0 +1,102 @@
+---
+title: Secondary DNS with DNSMadeEasy
+excerpt:
```

Comment:
How does "Secondary DNS can be complicated to set up. We've simplified things with provider specific settings for DNSMadeEasy." sound?